### PR TITLE
added ray data batch classification template 

### PR DESCRIPTION
**Pull request by soffer-anyscale**
Added the Ray Data Batch Inference template as part of the "Run Quickstart" project.

This project will test out linking the popular examples on the Ray Docs to hosted Anyscale templates so users can click a "Run Quickstart" button that will open up the template on hosted Anyscale.

This is also part of the OSS Ray/K8s -> Hosted Anyscale Funnel project.

This PR includes only one template so I can test out adding the templates. Shortly, we'd want at least the top 10 public examples to follow.

### CODE REVIEW

File: configs/ray-data-batch-classification/aws.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+head_node_type:
+  name: head
+  instance_type: g5.xlarge
+worker_node_types:
+- name: gpu_worker
+  instance_type: g5.xlarge
+  min_workers: 1
+  max_workers: 3
```
Comment (aslonnie) [inline configs/ray-data-batch-classification/aws.yaml:8]: add an eol at the end of this file

File: configs/ray-data-batch-classification/gcp.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+head_node_type:
+  name: head
+  instance_type: n1-standard-4-nvidia-t4-16gb-1
+worker_node_types:
+- name: gpu_worker
+  instance_type: n1-standard-4-nvidia-t4-16gb-1
+  min_workers: 1
+  max_workers: 3
```
Comment (aslonnie) [inline configs/ray-data-batch-classification/gcp.yaml:8]: and also here

File: .gitignore [hunk 3]
```diff
@@ -157,7 +157,7 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
+.idea/
```
Comment (aslonnie) [inline .gitignore:160]: why change this?

File: configs/ray-data-batch-classification/aws.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+head_node_type:
+  name: head
+  instance_type: g5.xlarge
```
Comment (shomilj) [inline configs/ray-data-batch-classification/aws.yaml:3]: Can this use the serverless auto select workers mode? We are trying to push away from GPU head nodes due to idle cost, especially when using these for community edition type use cases.

File: .gitignore (hunk 3 above)
Comment (soffer-anyscale) [reply .gitignore:160]: I was using PyCharm and it creates the .idea folder, we can keep it commented if needed.

File: configs/ray-data-batch-classification/aws.yaml (hunk 4 above)
Comment (soffer-anyscale) [reply configs/ray-data-batch-classification/aws.yaml:3]: Yes, definitely.

**Review by soffer-anyscale — COMMENTED** [review]
Updated files based on feedback.

File: configs/ray-data-batch-classification/aws.yaml (hunk 4 above)
Comment (aslonnie) [reply configs/ray-data-batch-classification/aws.yaml:3]: is this serverless now? seems that it is still using gpu head node.

Comment (aslonnie) [reply configs/ray-data-batch-classification/aws.yaml:3]: I think what @shomilj meant is to not use a gpu node as head node, and run workloads on worker nodes with auto gpu node selection.

**Review by aslonnie — COMMENTED** [review]
removing myself from review list. you need a template codeowner to review this.

**Review by scottsun94 — APPROVED** [review]

**Pull request merged**